server/models/studentModel.js

```diff
@@ -0,0 +1,39 @@
+
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema;
+
+const studentSchema = new Schema({
+
+    fName: {
+        type: String,
+        required: true
+
+    },
+    age: {
+        type: Number,
+        required: true
+
+    },
+    location: {
+        type: String,
+
+    },
+    sStatus: {
```

Comment:
is there a reason for it being sStatus instead of Status?

Reply:
same as happend with the name, and it means student status